pubtools/_quay/item_processor.py

```diff
@@ -186,7 +186,6 @@ def extract_manifests(
                     results.extend(ret)
                 else:
                     results.append(ret)
-                break
```

Comment:
This produces duplicated to be signed amd64 manfiests from processing both manifest list and v2 manifest.